ckanext/csrf_filter/anti_csrf.py

```diff
@@ -90,6 +91,9 @@ def configure(config):
     token_expiry_age = 60 * config.get('ckanext.csrf_filter.token_expiry_minutes', 30)
     token_renewal_age = 60 * config.get('ckanext.csrf_filter.token_renewal_minutes', 10)
 
+    exempt_rules = config.get('ckanext.csrf_filter.exempt_rules', None)
```

Comment:
`API_URL` should be subsumed into this list.

Reply:
`API_URL` value is now the default exempt rule. The custom ones from the config value will be added on if they exist